frontend/src/scenes/notebooks/Notebook/MentionsExtension.tsx

```diff
@@ -0,0 +1,205 @@
+        if (!query) {
+            return meFirstMembers
+        }
+        return fuse.search(query).map((result) => result.item)
+    }, [query, fuse])
+
+    useEffect(() => {
+        setSelectedIndex(0)
+        setSelectedHorizontalIndex(0)
+    }, [query])
+
+    const execute = async (member: OrganizationMemberType): Promise<void> => {
+        if (editor) {
+            editor
+                .deleteRange(range)
+                .insertContentAt(range.from, [
+                    {
+                        type: NotebookNodeType.Mention,
+                        attrs: {
+                            id: member.user.id,
+                        },
+                    },
+                ])
+                .run()
+
+            onClose?.()
+        }
+    }
+
+    const onPressEnter = async (): Promise<void> => {
+        const member = filteredMembers[selectedIndex]
+        await execute(member)
+    }
+    const onPressUp = (): void => {
+        setSelectedIndex(Math.max(selectedIndex - 1, -1))
+    }
+    const onPressDown = (): void => {
+        setSelectedIndex(Math.min(selectedIndex + 1, filteredMembers.length - 1))
+    }
+
+    const onKeyDown = useCallback(
+        (event: KeyboardEvent): boolean => {
+            const keyMappings = {
+                ArrowUp: onPressUp,
+                ArrowDown: onPressDown,
+                Enter: onPressEnter,
+            }
+
+            if (keyMappings[event.key]) {
+                keyMappings[event.key]()
+                return true
+            }
+
+            return false
+        },
+        [selectedIndex, selectedHorizontalIndex, filteredMembers]
+    )
+
+    // Expose the keydown handler to the tiptap extension
+    useImperativeHandle(ref, () => ({ onKeyDown }), [onKeyDown])
+
+    if (!editor) {
+        return null
+    }
+
+    return (
+        <div className="space-y-px">
+            {filteredMembers.map((member, index) => (
+                <LemonButton
+                    key={member.id}
+                    fullWidth
+                    status="primary-alt"
+                    icon={<ProfilePicture name={member.user.first_name} email={member.user.email} size="sm" />}
+                    active={index === selectedIndex}
+                    onClick={async () => await execute(member)}
+                >
+                    {`${member.user.first_name} <${member.user.email}>`}
+                </LemonButton>
+            ))}
+
+            {filteredMembers.length === 0 && (
+                <div className="text-muted-alt p-1">
+                    No member matching <code>@{query}</code>
+                </div>
+            )}
+        </div>
+    )
+})
+
+const MentionsPopover = forwardRef<MentionsRef, MentionsPopoverProps>(function MentionsPopover(
+    { visible = true, decorationNode, children, onClose, ...props }: MentionsPopoverProps,
+    ref
+): JSX.Element | null {
+    return (
+        <Popover
+            placement="right-start"
```

Comment:
Tiny thing but it felt strange for the popover to appear right next to the `@` symbol. Double checked GitHub and Slack which both go above or below but really up to you